pkg/providers/ovirt/mapper/mapper_test.go

```diff
@@ -40,6 +44,53 @@ var _ = Describe("Test mapping virtual machine attributes", func() {
 		bootloader := vmSpec.Spec.Template.Spec.Domain.Firmware.Bootloader.BIOS
 		Expect(bootloader).ToNot(BeNil())
 	})
+
+	It("For HA", func() {
```

Comment:
I think that Ginkgo expectation is that user defines what should happen in the `It` function text; i.e. `It("should map HA")` so it sounds naturally as "It should map HA".

Reply:
Yes, that looks better, thanks.